timesketch/lib/analyzers/sequence_sessionizer_test.py

```diff
@@ -0,0 +1,391 @@
+    event_seq_name = 'mock_seq_sessionizer'
+    return_fields = ['hostname', 'source_short', 'timestamp']
+    event_seq = [{
+        'hostname': 'host',
+        'source_short': 'FILE'
+    }, {
+        'hostname': 'host',
+        'source_short': 'WEBHIST'
+    }]
+
+
+class OneEventSequenceSessionizer(SequenceSessionizerSketchPlugin):
+    event_seq_name = 'one_event_seq_sessionizer'
+    return_fields = ['hostname', 'source_short', 'timestamp']
+    event_seq = [{'hostname': 'host', 'source_short': 'FILE'}]
+
+
+#list of dictionaries with attributes for mock events
+one_session_args = [{
+    'hostname': 'host',
+    'source_short': 'FILE'
+}, {
+    'hostname': 'host',
+    'source_short': 'WEBHIST'
+}]
+
+two_sessions_args = [{
+    'hostname': 'host',
+    'source_short': 'FILE'
+}, {
+    'hostname': 'host',
+    'source_short': 'WEBHIST'
+}, {
+    'hostname': 'host',
+    'source_short': 'FILE'
+}, {
+    'hostname': 'host',
+    'source_short': 'WEBHIST'
+}]
+
+one_event_args = [{'hostname': 'host', 'source_short': 'FILE'}]
+
+#max_time_diff_micros has the same value in the mock sequence sessionizers
+max_allowed = SequenceSessionizerSketchPlugin.max_time_diff_micros
+
+
+class TestSequenceSessionizerPlugin(BaseTest):
+    """Tests the functionality of the sequence sessionizing sketch analyzer."""
+    @mock.patch('timesketch.lib.analyzers.interface.ElasticsearchDataStore',
+                MockDataStore)
+    def test_sessionizer(self):
+        """Test basic sequence sessionizer functionality."""
+        index = 'test_index'
+        sketch_id = 1
+        sessionizer = ManyEventsSequenceSessionizer(index, sketch_id)
+        self.assertIsInstance(sessionizer, SequenceSessionizerSketchPlugin)
+        self.assertEqual(index, sessionizer.index_name)
+        self.assertEqual(sketch_id, sessionizer.sketch.id)
+
+    @mock.patch('timesketch.lib.analyzers.interface.ElasticsearchDataStore',
+                MockDataStore)
+    def test_one_session(self):
+        """Test one sequence of events is finded and allocated as a
+        session."""
+        with mock.patch.object(SequenceSessionizerSketchPlugin,
+                               'event_stream',
+                               return_value=_create_mock_event(
+                                   0,
+                                   2,
+                                   one_session_args,
+                                   time_diffs=[max_allowed])):
+            index = 'test_index'
+            sketch_id = 1
+            sessionizer = ManyEventsSequenceSessionizer(index, sketch_id)
+            message = sessionizer.run()
+            self.assertEqual(
+                message,
+                'Sessionizing completed, number of {0} session created: 1'.
+                format(sessionizer.event_seq_name))
+
+            ds = MockDataStore('test', 0)
+            event1 = (ds.get_event('test_index', '0', stored_events=True))
```

Comment:
Why the extra parentheses around the `ds.get_event()` call? I see this in many places in the code